projects/lib/src/board/westernboard.cpp

```diff
@@ -496,6 +510,15 @@ QString WesternBoard::castlingRightsString(FenNotation notation) const
 	return str;
 }
 
+bool WesternBoard::pawnAmbiguity( StepType t ) const
+{
+	int count = 0;
+	for (auto a: pawnSteps)
```

Comment:
ACK. Corrected. 

--- Remark
Actually the method name is not good: pawnAmbiguity is a name for a quantifying method.
I thought about various names isBla... or hasBla... or pawnHasBla... to indicate a boolean result. I finally changed the method. It will now give the number of possible steps of the given type. So the method vInitialize now tests whether there are more than one FreeSteps.